src/extension/ddtrace.c

```diff
@@ -121,6 +121,8 @@ static int _ddtrace_rshutdown_testing(SHUTDOWN_FUNC_ARGS)
 
 const char *nullable dd_trace_version() { return _mod_version; }
 
+bool dd_trace_is_loaded() { return _mod_version != NULL; }
```

Comment:
It's an odd choice to use the `_mod_version` instead of a dedicated variable, for example the `_mod_version` can be `NULL` for legitimate reasons (https://heap.space/xref/PHP-7.3/Zend/zend_API.c?r=e9481888#3690)

Reply:
Fair point